src/app.js

```diff
@@ -182,9 +182,6 @@ var run = function () {
   var editor = new Editor(document.getElementById('input'))
 
   // ---------------- FilePanel --------------------
-  // TODO: All FilePanel related CSS should move into file-panel.js
-  // app.js provides file-panel.js with a css selector or a DOM element
-  // and file-panel.js adds its elements (including css), see "Editor" above
   var css = csjs`
     .filepanel-container    {
```

Comment:
Regarding css.
I recommend to read:
* https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660

Despite that - the separation of css/html/js in separate files is considered - not separation of concerns, but separation of technology. The markup of a component (e.g. a <div class="btn">...) and the styling of it (e.g. .btn { ... }) and the javascript (e.g. document.querySelector('.btn').onclick = ...) will always be highly coupled anyway.

The traditional way to deal with this is to have 3 files (css, html and js)
All very long.
You scroll through many lines of css to find the `.btn { ... }` related css
You scroll through many lines of html to find the `<div class="btn">...`
You scroll through many lines of js to find the `...querySelector('.btn').addEventListner...`

Modern practice is to make self contained components, that include all the css/js/html necessary for one e.g. button component.

So if a change needs to be done to the button, one file or one place contains all button related code html, css and javascript.

It reduces the cognitive load significantly.
Also it helps A LOT with the isolation of components. `csjs` suffixes classnames, so that the styling does not clash with the styling of other components - which can introduce very complicated layout bugs.

I would strongly recommend to not use global variables in css (=global css selectors) and follow the best practice of all programming language that usually consider global variables to be a bad thing.